python/taichi/core/util.py

```diff
@@ -285,6 +287,7 @@ def get_unique_task_id():
     import_ti_core()
     if get_os_name() != 'win':
         dll = ctypes.CDLL(get_core_shared_object(), mode=ctypes.RTLD_LOCAL)
+        ti_core.set_tmp_dir(prepare_sandbox())
```

Comment:
I don't understand this line - could you add some comments here?

Reply:
How about this: revert all changes in `core/util.py`, fix C backend tmp_dir in release mode iapr so this could be merged quicker?

Reply:
It actually looks better now. I think we can do it in this PR.